librz/util/subprocess.c

```diff
@@ -19,6 +19,9 @@ struct rz_subprocess_t {
 };
 
 static volatile long pipe_id = 0;
+static DWORD mode_stdin;
+static DWORD mode_stdout;
+static DWORD mode_stderr;
```

Comment:
i'm not a big fan of global variables. do you think is possible to avoid this and have local ones? after all these are only used when subprocess is executed.

Reply:
There are in general some global vars in this file that can't be removed because they are functional to the way rzsubprocess works. However, it remains to see whether these vars could be part of the `rz_subprocess_t` instead of globals. That I don't know.

Reply:
I would say if the variable contains state directly related to the global state of the OS running the process, then it is fine to have global vars since only one instance of them can be valid anyway. But depends on the case.

Reply:
It's ok as globals then ;)